e2e/common/common.go

```diff
@@ -68,19 +75,29 @@ func NewConfig(
 	conf := lib.Config{
 		Name:        name,
 		BaseDir:     basedirAbs,
-		LoomPath:    loompathAbs,
+		LoomPath:    "error",
```

Comment:
Its a debug thing, I'll remove it; The value set in NewConfig should be unused.
Basically, there are two loom paths, so the value actually used is going to be the one set in the node. We still need Config to have a loomPath member because of the way ["text/template" works with lib.Config](https://github.com/loomnetwork/loomchain/blob/issue1325-e2e/e2e/engine/cmd.go#L56).